util/esclient.go

```diff
@@ -24,30 +23,22 @@ var (
 // GetClient7 returns the es7 client
 func GetClient7() *es7.Client {
 	// initialize the client if not present
-	if client7 == nil {
-		clientInit.Do(func() {
-			initClient7()
-		})
-	}
+	clientInit.Do(func() {
```

Comment:
@bietkul You removed the nil check instead of removing the `clientInit.Do(..)` call.